scripts/saveImageDay.sh

```diff
@@ -1,30 +1,33 @@
 #!/bin/bash
-source $ALLSKY_HOME/config.sh
-source $ALLSKY_HOME/scripts/filename.sh
-source $ALLSKY_HOME/scripts/darkCapture.sh	# does not return if in darkframe mode
-source $ALLSKY_HOME/scripts/ftp-settings.sh
+
+ME="$(basename "$BASH_ARGV0")"
+
+source "${ALLSKY_HOME}/variables.sh"
+source "${ALLSKY_CONFIG}/config.sh"
+source "${ALLSKY_SCRIPTS}/filename.sh"
+source "${ALLSKY_SCRIPTS}/darkCapture.sh"	# does not return if in darkframe mode
+source "${ALLSKY_SCRIPTS}/ftp-settings.sh"
 
 cd $ALLSKY_HOME
-ME="$(basename "$BASH_ARGV0")"	# Include script name in output so it's easier to find in the log file
 
 IMAGE_TO_USE="$FULL_FILENAME"
 # quotes around $IMAGE_TO_USE below, in case it has a space or special characters.
 
-# Make sure the image isn't corrupted
+# Quick check to make sure the image isn't corrupted.
 identify "$IMAGE_TO_USE" >/dev/null 2>&1
 RET=$?
 if [ $RET -ne 0 ] ; then
-	echo "*** $ME: ERROR: Image '${IMAGE_TO_USE} is corrupt; ignoring."
-	exit 1
+	echo "${RED}*** $ME: ERROR: Image '${IMAGE_TO_USE} is corrupt; ignoring.${NC}"
+	exit 3
 fi
 
 # Resize the image if required
 if [[ $IMG_RESIZE == "true" ]]; then
         convert "$IMAGE_TO_USE" -resize "$IMG_WIDTH"x"$IMG_HEIGHT" "$IMAGE_TO_USE"
 	RET=$?
 	if [ $RET -ne 0 ] ; then
-		echo "*** $ME: ERROR: IMG_RESIZE failed with RET=$RET"
-		exit 1
+		echo "${RED}*** $ME: ERROR: IMG_RESIZE failed with RET=$RET${NC}"
```

Comment:
What's the user supposed to do with `$RET`? I don't see any description of return codes in the `convert(1)` manpage, and there should be a pretty good clue about what went wrong from `convert`.